smartcontractkit/external-adapters-js
# sAVAX EA

## Closes 30325

## Description

This PR implements the new composite EA to pull the price of `sAVAX-USD`.  The price is derived by:

1) Fetching the `AVAX-USD` price from a data provider
2) Get the number of `pooledAvaxByShares` by calling the `getPooledAvaxByShares` function from this contract https://snowtrace.io/address/0x2b2c81e08f1af8835a78bb2a90ae924ace0ea4be#readProxyContract
3) Multiplying the two

## Changes

- Implement new `savax-price`composite EA
- Write tests for EA



## Steps to Test

1. Run unit/integration/e2e tests and verify that they pass
2. Run EA by following instructions in ReadME
3. Verify that the EA returns a successful response
4. Verify that the response is correct by seeing if the result is the product of the `AVAX-USD` price returned from a data provider and the `getPooledAvaxByShares` function

## Quality Assurance
